src/microcosm.js

```diff
@@ -394,7 +394,7 @@ class Microcosm extends Emitter implements Domain {
    * The default registration method for Microcosms
    */
   register() {
-    return {}
+    return null
```

Comment:
Abysmally low memory savings, but also avoiding all of the checks to see if the registration supports a specific action.